django/thunderstore/repository/models/package.py

```diff
@@ -44,6 +44,12 @@ class Package(models.Model):
         on_delete=models.PROTECT,
         related_name="owned_packages",
     )
+    namespace = models.ForeignKey(
```

Comment:
What is populating namespace for new packages? Seems to be lacking entirely from this PR